release_notes/368-release-notes.adoc

```diff
@@ -7,7 +7,9 @@ toc::[]
 
 {product-title} ({product-title-short}) 3.68 includes feature enhancements, bug fixes, scale improvements, and other changes.
 
-*Release date:* February 2, 2022
+*3.68 Release date:* February 2, 2022 +
+*3.68.1 Release date:* February 14, 2022 +
+*3.68.2 Release date:* June 14, 2022
```

Comment:
WDYT about a bulleted list for these entries? Might improve readability as the list gets longer.